package.json

```diff
@@ -2,14 +2,14 @@
   "name": "rebass",
   "version": "3.0.0-0",
   "description": "React primitive UI components built with styled-system",
-  "main": "dist/index.js",
-  "module": "dist/index.es.js",
+  "main": "dist/index",
   "sideEffects": false,
   "scripts": {
-    "prepare": "rm -rf dist && npm run build && npm run build:es && npm run build:emotion",
-    "build": "NODE_ENV=cjs babel src -d dist",
-    "build:es": "NODE_ENV=es babel src/index.js -o dist/index.es.js",
-    "build:emotion": "NODE_ENV=emotion babel src -o dist/emotion.js",
+    "prepare": "rm -rf dist && mkdir dist && npm run build:cjs && npm run build:esm && npm run build:emotion:cjs && npm run build:emotion:esm",
+    "build:cjs": "NODE_ENV=cjs babel src/index.js -o dist/index.js",
+    "build:esm": "NODE_ENV=esm babel src/index.js -o dist/index.mjs",
```

Comment:
Does `.mjs` work with webpack and rollup? I thought that was only for Node.js

Reply:
Both have priority to resolve mjs first
https://github.com/rollup/rollup-plugin-node-resolve/blob/master/src/index.js#L11
https://github.com/webpack/webpack/blob/master/lib/WebpackOptionsDefaulter.js#L327

Reply:
That's certainly simpler, thanks!